README.md

```diff
@@ -13,9 +13,19 @@ concurrent and utilized BE systems. It's often the case that high latency or big
 CPU usage is caused by very specific requests that are triggering
 inefficient code. Finding this code is usually pretty difficult.
 
-## How to use it
+## How does it look like
```

Comment:
[minor suggestion for the future] I would name this paragraph simply "Demo". For me more intuitive.